File: 백준/Bronze/10101. 삼각형 외우기/삼각형 외우기.py
angle = []

for i in range(3):
    angle.append(int(input()))

if angle.count(60) == 3:
    print('Equilateral')
elif sum(angle) == 180 and len(set(angle)) == 2:
    print('Isosceles')
elif sum(angle) == 180 and len(set(angle)) == 3:
    print('Scalene')
elif sum(angle) != 180:
    print('Error')

File: 백준/Bronze/10101. 삼각형 외우기/삼각형 외우기.cc
#include <iostream>
using namespace std;

int main() {
  ios::sync_with_stdio(false);
  cin.tie(NULL);

  int x, y, z;
  cin >> x >> y >> z;
  
  if (x + y + z != 180) {
    cout << "Error" << '\n';
  }
  else if (x == y && y == z) {
    cout << "Equilateral" << '\n';
  }
  else if (x == y || x == z || y == z) {
    cout << "Isosceles" << '\n';
  }
  else {
    cout << "Scalene" << '\n';
  }

  return 0;
}